packages/backend/src/modules/tracked-txs/modules/l2-costs/repositories/L2CostsRepository.ts

```diff
@@ -53,6 +53,22 @@ export class L2CostsRepository extends BaseRepository {
     return rows.map((r) => toRecord(r))
   }
 
+  async getByProjectSinceTimestamp(
```

Comment:
maybe `async getByProjectAndTimeRange(projectId: ProjectId, timeRange: [UnixTime, UnixTime]) {...}` then `const [from, to] = timeRange`